test/runtests.py

```diff
@@ -99,9 +103,19 @@ def test_path(folder, is_file):
                 baseline = os.path.splitext(js_file)[0] + '.baseline'
                 baseline = os.path.join(folder, baseline)
                 if os.path.isfile(baseline):
-                    expected_output = None
+                    baseline_output = None
                     with open(baseline, 'r') as bs_file:
-                        expected_output = bs_file.read().replace('\r', '')
+                        baseline_output = bs_file.read()
+
+                    # clear carriage return
+                    # todo: remove carriage return at the end of the line
+                    #       or better fix ch to output same on all platforms
+                    expected_output = baseline_output.replace('\r', '')
+
+                    # normalize tabs on both outputs to 2 spaces
+                    expected_output = expected_output.replace('\t', '  ')
```

Comment:
Since the flag is in test harness (runtests.cmd/rl.exe), this script can be compatible and provide the flag.

Now I think on it, the history of this flag might be: We used to always print `\t` here, so all our baselines contain `\t`. But then we wanted to be compatible and use 3 spaces instead. To avoid updating all baselines, we chose this test flag to keep printing `\t` in all tests. (I think we should change this, no point to print `\t`. Should update all baselines to 3 spaces.)